### PR TITLE
Jodit 3.3.2

### PR DESCRIPTION
**Pull request by xdan**
# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required):
* Official website (optional, not the repository):
* NPM package url (optional):
* License and its reference:
* GitHub / Bitbucket popularity (required):
  * Count of watchers:
  * Count of stars:
  * Count of forks:
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [ ] This lib was not found on cdnjs repo
* [ ] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [ ] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jodit/package.json [hunk 1]
```diff
@@ -10,18 +10,36 @@
...
     "jodit"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "MIT",
   "author": "Chupurnov <chupurnov@gmail.com> (http://xdsoft.net/)",
   "repository": {
     "type": "git",
     "url": "https://github.com/xdan/jodit.git"
   },
+  "autoupdate": {
```
Comment (MattIPv4) [inline ajax/libs/jodit/package.json]: Why have you added git auto-update when the npm auto-update config already exists?

File: ajax/libs/jodit/package.json [hunk 2]
```diff
@@ -10,18 +10,36 @@
...
+    ]
+  },
   "npmName": "jodit",
   "npmFileMap": [
     {
-      "basePath": "",
+      "basePath": "build",
       "files": [
-        "*jodit*.+(css|js|map)"
+        "jodit.min.js",
```
Comment (MattIPv4) [inline ajax/libs/jodit/package.json]: Why only include the minified files, would it not be sensible to also include the unminified files as well?

Comment (MattIPv4) [inline ajax/libs/jodit/package.json]: Using a glob of [`jodit*.+(css|js)` would include all those files](https://www.digitalocean.com/community/tools/glob?glob=jodit%2A.%2B%28css%7Cjs%29&tests=jodit.css&tests=jodit.es2018.css&tests=jodit.es2018.js&tests=jodit.es2018.min.css&tests=jodit.es2018.min.js&tests=jodit.js&tests=jodit.min.css&tests=jodit.min.js) or you could use a glob of [`jodit*.min.+(css|js)` to include only the minified files](https://www.digitalocean.com/community/tools/glob?glob=jodit%2A.min.%2B%28css%7Cjs%29&tests=jodit.css&tests=jodit.es2018.css&tests=jodit.es2018.js&tests=jodit.es2018.min.css&tests=jodit.es2018.min.js&tests=jodit.js&tests=jodit.min.css&tests=jodit.min.js).

File: ajax/libs/jodit/package.json [hunk 3]
```diff
@@ -10,18 +10,36 @@
     "rich",
     "jodit"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "MIT",
```
Comment (MattIPv4) [inline ajax/libs/jodit/package.json]:
```suggestion
  "license": "GPL-2.0-or-later",
```

The main license in the source repository appears to be GNU GPL 2.0?

File: ajax/libs/jodit/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jodit",
-  "version": "2.5.62",
+  "version": "3.3.2",
```
Comment (MattIPv4) [inline ajax/libs/jodit/package.json]: Please include all missing versions from the CDN between 3.2.65 ([last version in cdnjs](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/jodit/)) & 3.3.2

Comment (xdan) [reply ajax/libs/jodit/package.json]: Should I add several PR for all missing versions? @MattIPv4

File: ajax/libs/jodit/package.json (hunk 3 above)
Comment (xdan) [reply ajax/libs/jodit/package.json]: Main MIT @MattIPv4

File: ajax/libs/jodit/package.json (hunk 4 above)
Comment (MattIPv4) [reply ajax/libs/jodit/package.json]: They can all be in this PR.

File: ajax/libs/jodit/package.json (hunk 3 above)
Comment (MattIPv4) [reply ajax/libs/jodit/package.json]: Ah, cool.

File: ajax/libs/jodit/package.json (hunk 4 above)
Comment (xdan) [reply ajax/libs/jodit/package.json]: Before 3.3.2 was only one release 3.3.1, I added files

File: ajax/libs/jodit/package.json (hunk 1 above)
Comment (xdan) [reply ajax/libs/jodit/package.json]: Removed

File: ajax/libs/jodit/package.json (hunk 2 above)
Comment (xdan) [reply ajax/libs/jodit/package.json]: Added *jodit*.+(css|js|map)

Comment (xdan) [reply ajax/libs/jodit/package.json]: Added *jodit*.+(css|js|map)

File: ajax/libs/jodit/package.json [hunk 5]
```diff
@@ -19,7 +19,7 @@
   "npmName": "jodit",
   "npmFileMap": [
     {
-      "basePath": "",
+      "basePath": "build",
       "files": [
         "*jodit*.+(css|js|map)"
```
Comment (MattIPv4) [inline ajax/libs/jodit/package.json]: This would also match the unminified files that are published, these should be included in the PR please :)

Comment (xdan) [reply ajax/libs/jodit/package.json]: Added

**Comment by xdan** [comment]
@MattIPv4 What is required of me for PR?

**Comment by xdan** [comment]
@xtuc Done

**Comment by xdan** [comment]
I do not know but Jodit 3.3. already in cdnjs, I think this PR can be closed. 
https://cdnjs.com/libraries/jodit

File: ajax/libs/jodit/package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jodit",
-  "version": "3.2.65",
+  "version": "3.3.2",
```
Comment (MattIPv4) [inline ajax/libs/jodit/package.json]: This change needs to be reverted please
```suggestion
  "version": "3.2.65",
```

**Comment by MattIPv4** [comment]
If you can resolve the merge conflict and revert any change to the version, this PR is still useful as it provides a better auto-update file map and corrects the license.

File: ajax/libs/jodit/package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "jodit",
-  "version": "3.3.24",
+  "version": "3.2.65",
```
Comment (MattIPv4) [inline ajax/libs/jodit/package.json]: Please revert this version change :)

**Pull request merged**